custom.conf.template

```diff
@@ -0,0 +1,8 @@
+[Labels]
+required-labels-any: Either provide a list of comma-separated labels, or leave blank if not using
```

Comment:
Might be best to have the comments as `# comments` so that these config options default to being empty.